README.md

```diff
@@ -127,48 +131,56 @@ IAMユーザには以下の認可ポリシーが必要です。
 
 ## Usage
 
-`AwsSsmEnv#load`に渡すオプションの説明です。
+A description of the options passed to `AwsSsmEnv#load`.
 
 ### decryption: [Boolean]
 
-SecureStringのパラメータを復号化するかどうかを表すフラグ。  
-`true`を指定した場合は取得したSecureStringパラメータの値は復号化されている。  
-`false`の場合は暗号化されたまた環境変数値として設定される。  
-なお、このためのgemなのでデフォルトは`true`(復号化する)。
+Flag indicating whether to decrypt SecureString parameters.  
+If `true` is specified, the value of the acquired SecureString parameter is decrypted.  
+In case of `false` it is set as encrypted and environment variable value.  
+Since it is a gem for this, the default is `true` (decrypt).
 
 ### overwrite: [Boolean]
 
-すでに設定されている環境変数を上書きするかどうかを指定する。  
-`true`を指定した場合、環境変数が設定されていても取得したパラメータ値で上書きする。  
-`false`を指定した場合はすでに設定されている環境変数を上書きしない。  
-デフォルトは`false`(上書きしない)。  
-なお、`AwsSsmEnv#load!`を実行した場合、このフラグは自動的に`true`になる。
+Specify whether to overwrite an already set environment variable.  
+If `true` is specified, even if the environment variable is set,
+it overwrites it with the acquired parameter value.  
+If `false` is specified, do not overwrite already set environment variables.  
+The default is `false` (do not overwrite).  
+If you invoke `AwsSsmEnv#load!`, This flag will automatically be set to `true`.
 
 ### client: [Aws::SSM::Client]
 
-`Aws::SSM::Client`のインスタンスを指定する。  
-すでに生成済みのインスタンスがある場合にそれを設定するためのオプション。  
-生成済みのインスタンスがない場合は`ssm_client_args`を利用する。
+Specify an instance of `Aws::SSM::Client`.  
+An option to set it if there are already created instances.  
+If there are no instances already created use `ssm_client_args`.
```

Comment:
It's a bit difficult to understand.
I think `already` is verbose here.
And we should add `,` to split these two sentences (if I am right):

> If there are no instances created, use `ssm_client_args` instead.